### DISCUSSION TITLE
Nextcloud Desktop App

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#37] [General]
# Nextcloud Desktop App

Check if Nextcloud Desktop App is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/c65aa862-ceb8-4d6c-b1e4-985680da9044

**Comment by OpenSource03** [comment]
More info:
https://help.nextcloud.com/t/windows-10-arm64-client-support/104236/7
